src/index.html

```diff
@@ -5,9 +5,53 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&   display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="container">
+      <div class="container__card card"
+          data-qa="card">
+        <img class="card__picture picture"
+          src="images/imac.jpeg"
+          alt="Apple A1419 iMac 27'' Retina 5K Monoblock"
+        >
```

Comment:
Need to check and fix it everywhere
<img width="1074" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/64750958/f9fe1947-cc8c-4b27-9514-88204f295290">